### ISSUE TITLE
Feature Request: Add a flag to disable handling of window close events

### ISSUE THREAD

**Issue by asdfjkluiop** [Plagman/gamescope#641]
I recently updated gamescope and was sort of disappointed that it now handles window close events. Is there any chance a flag could be added to allow disabling this? I'd be willing to write a PR for it if it's something that would be accepted.

**Comment by Joshua-Ashton** [comment]
Not super sure what you mean, like pressing the X on Gamescope?

**Comment by asdfjkluiop** [comment]
Correct, I mean making a flag that allows commit b5aa05ae8f772391bad7c352a2153b925286e8a7 to be optional. I'm on sway and I can't count the number of times I've accidentally closed a game because my brain thought mod+shift+q was how I get out of fullscreen lol. Gamescope fixed this issue nicely by ignoring window close events however that's been changed. Clearly most people probably want it to handle them properly but I think it'd be nice to have a flag that allows said events to be ignored.

**Comment by dafta** [comment]
https://xkcd.com/1172/

**Comment by asdfjkluiop** [comment]
> https://xkcd.com/1172/

LOL, yes, basically...although I think this is a little less "horrifying." Before gamescope I used to use a sway mode to disable mod+shift+q, gamescope fixed that for me, now I have my problem back...an option to disable the events would be nice. For now I've just patched out the events myself, at some point I'll write a PR to add an actual flag for it...provided the maintainers are open to it.
